src/utils/faram.js

```diff
@@ -5,7 +5,6 @@ export const alterResponseErrorToFaramError = (e) => {
         errors = e.reduce(
             (acc, error) => {
                 const {
-                    internalNonFieldErrors = [],
```

Comment:
Why removed this line?

Reply:
Was not being used

Reply:
This was used to capture internalNonFieldErrors